packages/puppeteer-core/src/common/PDFOptions.ts

```diff
@@ -219,18 +219,56 @@ export type ParsedPDFOptions = Required<
 
 /**
  * @internal
+ *
+ * @remarks All A series paper format sizes in inches are calculated from centimeters
+ * rounded mathematically to four decimal places.
  */
-export const paperFormats: Record<LowerCasePaperFormat, PaperFormatDimensions> =
-  {
-    letter: {width: 8.5, height: 11},
-    legal: {width: 8.5, height: 14},
-    tabloid: {width: 11, height: 17},
-    ledger: {width: 17, height: 11},
-    a0: {width: 33.1102, height: 46.811},
-    a1: {width: 23.3858, height: 33.1102},
-    a2: {width: 16.5354, height: 23.3858},
-    a3: {width: 11.6929, height: 16.5354},
-    a4: {width: 8.2677, height: 11.6929},
-    a5: {width: 5.8268, height: 8.2677},
-    a6: {width: 4.1339, height: 5.8268},
-  } as const;
+export const paperFormats: Record<
+  LowerCasePaperFormat,
+  Record<'cm' | 'in', PaperFormatDimensions>
+> = {
+  letter: {
+    cm: {width: 21.59, height: 27.94},
+    in: {width: 8.5, height: 11},
+  },
+  legal: {
+    cm: {width: 21.59, height: 35.56},
+    in: {width: 8.5, height: 14},
+  },
+  tabloid: {
+    cm: {width: 27.94, height: 43.18},
+    in: {width: 11, height: 17},
+  },
+  ledger: {
+    cm: {width: 43.18, height: 27.94},
+    in: {width: 17, height: 11},
+  },
+  a0: {
+    cm: {width: 118.9, height: 84.1},
+    in: {width: 46.811, height: 33.1102},
+  },
+  a1: {
+    cm: {width: 84.1, height: 59.4},
+    in: {width: 33.1102, height: 23.3858},
+  },
+  a2: {
+    cm: {width: 59.4, height: 42},
+    in: {width: 23.3858, height: 16.5354},
+  },
+  a3: {
+    cm: {width: 42, height: 29.7},
+    in: {width: 16.5354, height: 11.6929},
+  },
+  a4: {
+    cm: {width: 29.7, height: 21},
+    in: {width: 8.27, height: 11.7},
+  },
+  a5: {
+    cm: {width: 21, height: 14.8},
+    in: {width: 8.2677, height: 5.8268},
+  },
+  a6: {
+    cm: {width: 14.8, height: 10.5},
+    in: {width: 5.8268, height: 4.1339},
```

Comment:
Some of these values seemed flipped, other seem to be changed for the `in`.
Can you please revert all changes to the `in` parts, I believe all of them should be correct.
The width should always (except at `ledger`) be the smaller of the two, and people can use the `landscape` prop if they want another view.

Reply:
Right, my bad.

Do you want me to update my previous commit or create a fix-up commit or a separate `fix` commit?

Reply:
It does not matter, we `squash & merge` our PRs, so however you prefer.